test_other_bounds_low_res fails on Windows

```
    def test_other_bounds_low_res(records):
        result = find_grid_duplicates(
            records,
            "scientificName",
            resolution=0.1333333402874744,
            bounds=(-78.9909352282, -4.29818694419, -66.8763258531, 12.4373031682),
        )
        expected = pd.Series(
            [
                True,
                True,
                True,
                True,
                True,
                True,
                True,
                False,
                False,
                True,
                True,
                True,
                False,
                False,
                True,
                True,
                True,
                False,
                True,
                False,
                False,
                True,
            ]
        )
>       pd.testing.assert_series_equal(result, expected, check_dtype=False)

tests\geographic\duplicates\test_find_grid_duplicates.py:77:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pandas\_libs\testing.pyx:53: in pandas._libs.testing.assert_almost_equal
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   AssertionError: Series are different
E
E   Series values are different (9.09091 %)
E   [index]: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21]
E   [left]:  [True, True, True, True, True, True, True, False, False, True, True, True, False, False, True, True, True, False, nan, False, False, nan]
E   [right]: [True, True, True, True, True, True, True, False, False, True, True, True, False, False, True, True, True, False, True, False, False, True]

pandas\_libs\testing.pyx:168: AssertionError
```

--- a/regi0/geographic/duplicates.py
+++ b/regi0/geographic/duplicates.py
@@ -53,7 +53,12 @@ def _create_id_grid(
     height = np.ceil((ymax - ymin) / resolution).astype(int)
     width = np.ceil((xmax - xmin) / resolution).astype(int)
     transform = rasterio.transform.from_origin(xmin, ymax, resolution, resolution)
-    arr = np.arange(height * width, dtype=np.uint32).reshape(height, width)
+
+    # The unique ID values to fill the grid with start from 1 to make sure
+    # the value 0 is reserved for no-data and that the whole numeric range
+    # of a 32-bit unsigned integer is available for IDs (in the very
+    # unlikely case this is needed).
+    arr = np.arange(1, (height * width) + 1, dtype=np.uint32).reshape(height, width)
 
     memfile = rasterio.MemoryFile()
     grid = memfile.open(
@@ -64,6 +69,7 @@ def _create_id_grid(
         crs=crs,
         transform=transform,
         dtype=rasterio.uint32,
+        nodata=0,
     )
     grid.write(arr, 1)
 
@@ -116,7 +122,7 @@ def find_grid_duplicates(
         bounds = gdf.geometry.total_bounds
     grid = _create_id_grid(*bounds, resolution, gdf.crs.srs)
     gdf["__grid_id"] = rasterstats.point_query(
-        gdf, grid.read(1), affine=grid.transform, interpolate="nearest", nodata=-9999
+        gdf, grid.read(1), affine=grid.transform, interpolate="nearest", nodata=0
     )
     result = gdf.duplicated(subset=[species_col, "__grid_id"], keep=keep)
 

--- a/tests/geographic/duplicates/test_find_grid_duplicates.py
+++ b/tests/geographic/duplicates/test_find_grid_duplicates.py
@@ -1,6 +1,7 @@
 """
 Test cases for the regi0.geographic.duplicates.find_grid_duplicates function.
 """
+import numpy as np
 import pandas as pd
 
 from regi0.geographic.duplicates import find_grid_duplicates
@@ -68,10 +69,10 @@ def test_other_bounds_low_res(records):
             True,
             True,
             False,
-            True,
+            np.nan,
             False,
             False,
-            True,
+            np.nan,
         ]
     )
     pd.testing.assert_series_equal(result, expected, check_dtype=False)

